test/suite/autobinding.test.ts

```diff
@@ -439,7 +438,7 @@ suite('Auto Binding Test Suite', () => {
   });
 });
 
-async function deleteSettingsFiles() {
```

Comment:
the temporary settings files need to be deleted not to pollute the state for upcoming test suites. At the same time, binding needs to be cleaned up after each test.

Possibly there used to be a race condition between the two actions. This change aims to mitigate this.